go.mod

```diff
@@ -35,11 +35,8 @@ replace go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.open
 
 replace github.com/jackc/pgx/v4 => github.com/jackc/pgx/v5 v5.5.4
 
-// This replace directive supports the backlevel go version required by the ODH build
-replace github.com/project-codeflare/appwrapper v0.27.0 => github.com/project-codeflare/appwrapper v0.27.1-0.20241105160939-f71c6326acce
-
-// These replace directives deal with the backlevel ODH kueue version
-replace sigs.k8s.io/kueue v0.8.3 => github.com/opendatahub-io/kueue v0.7.0-odh-2
```

Comment:
you probably still need a replace directive for kueue, just to a different odh version.

Reply:
ok, will try

Reply:
Done